Bio/SubsMat/__init__.py

```diff
@@ -313,8 +312,18 @@ def print_full_mat(self, f=None, format="%4d", topformat="%4s",
                       " please use\n"
                       " print(mat.format(<arguments>, full=True)",
                       BiopythonDeprecationWarning)
+        if factor == 1:
+            mat = self
+        else:
+            mat = factor * self
+            warnings.warn("""
+Instead of
+mat.print_full_mat(..., factor, ...)
+please use
+mat2 = factor * mat
+print(mat2.format(..., full=True))""", BiopythonDeprecationWarning)
```

Comment:
This will put an unwanted leading new line in the message, won't it? How about:

```python
            warnings.warn("Instead of 'mat.print_full_mat(..., factor, ...)' please "
                          "use 'mat2 = factor * mat; print(mat2.format(..., full=True))'",
                          BiopythonDeprecationWarning)
```

Reply:
Fixed